### ISSUE TITLE
Include standard column breaks

### ISSUE THREAD

**Issue by pabuta88** [ThinkR-open/remedy#53]
Hi,

First of all, thanks a lot for your package, which simplifies working with RMarkdown a lot. 

I was wondering whether it would be possible to include similar to the xaringan pull-left and pull-right template a standard way of inserting a column break.

I prefer using ioslides and hence I need to wrap my markdown text and my chunks into 
<div id="left"> ... </div> and <div id="right"> ... </div>, respectively (setting the id the css file). I am not sure yet what would be the best way to include this feature but I guess many people work with separated columns and this would save them a lot of time.

What do you think?

Best, Pascal

**Comment by pabuta88** [comment]
Thank you for your quick reply. I am actually not yet sure what the best way would be to implement it. At the moment, I have something simple in mind like the following

Before:

```{r}
data(iris)
```

or

* This is right

After: 

<div class="left">
```{r}
data(iris)
```
</div>

and 

 <div class="right">
     * This is right
 </div>

The styling could then be done in the css file. What do you think? Of course it would be nice to have a mask to enter style details like width and float. But I think the most important part would be to wrap it in div-tags (similar to your example of turning selected text into a chunk).

**Comment by yonicd** [comment]
in general terms an addin that wraps text in the following manner could be possible, would it answer your needs?

```html
<div class="CLASSNAME">
  [TEXT]
 </div>
```

**Comment by pabuta88** [comment]
exactly. how would you let the user to choose the classname in the addin?
